internals/cli/cmd_run.go

```diff
@@ -275,3 +283,109 @@ func convertArgs(args [][]string) (map[string][]string, error) {
 
 	return mappedArgs, nil
 }
+
+func maybeCopyPebbleDir(srcDir, destDir string) error {
+	if srcDir == "" {
+		return nil
+	}
+	dirEnts, err := os.ReadDir(destDir)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		return err
+	} else if len(dirEnts) != 0 {
+		// Skip non-empty dir.
+		return nil
+	}
+	fsys := os.DirFS(srcDir)
+	// TODO: replace with os.CopyFS
+	err = _go_os_CopyFS(destDir, fsys)
```

Comment:
Nice, much simpler!

Naming nit: can we please call this `osCopyFS` or even just `copyFS`? Those underscores make it not look like Go somehow. :-)